scripts/fromCMGTools/templates/index.php

```diff
@@ -0,0 +1,131 @@
+<html>
```

Comment:
Actually you could just replace the index.php file in wremnants/Templates with this one, which I think is a bit better than what we have there

Reply:
Updated with scripts/fromCMGTools/templates/index.php